### ISSUE TITLE
Label overlaps with lake labels

### ISSUE THREAD

**Issue by imagico** [der-stefan/OpenTopoMap#197]
You might have already seen this - at the moment lake labels frequently overlap with other labels:

https://opentopomap.org/#map=12/52.2257/13.6895

**Comment by der-stefan** [comment]
Yes, there are several text and symbol layers and one of them deletes the label cache (since some symbols should be overlappable by texts...) But yes, the label layers could be re-sorted to prevent this.
